draft-ietf-taps-interface.md

```diff
@@ -1801,11 +1801,15 @@ Protocol Stack(s) backing a Connection. These Connection Properties may be
 Generic, applying regardless of transport protocol, or Specific, applicable to a
 single implementation of a single transport protocol stack. Generic Connection
 Properties are defined in {{connection-props}} below. 
-
+ 
 Protocol Specific Properties are defined in a transport- and 
-implementation-specific way, and MUST NOT apply across different protocols. 
-Too much reliance by an application on Protocol Specific Properties can significantly 
-reduce the flexibility of a transport services implementation.
+implementation-specific way to
+permit more specialized protocol features to be used. 
+Too much reliance by an application on Protocol Specific Properties can significantly reduce the flexibility 
+of a transport services implementation to make appropriate
+selection and configuration choices. Therefore, it is RECOMMENDED that
+Protocol Properties are normnally used for properties common across different protocols and that
```

Comment:
Typo in "normally", but: wouldn't this read better by altogether dropping the "normally"?

Reply:
Much better without the word! (I will remove)